test/test_manifest.json

```diff
@@ -1360,6 +1360,12 @@
        "enableXfa": true,
        "type": "eq"
     },
+    {  "id": "Bug1743245_example",
+       "file": "pdfs/Bug1743245_example.pdf",
```

Comment:
```javascript
    {  "id": "bug1743245",
       "file": "pdfs/bug1743245.pdf",
```
and please re-name the PDF file accordingly.

Also, you're missing an entry in the https://github.com/mozilla/pdf.js/blob/master/test/pdfs/.gitignore file (the `add_test.js` script should've updated that one as well).